addon.xml

```diff
@@ -3,6 +3,7 @@
     <requires>
         <import addon="xbmc.python" version="2.25.0" />
         <import addon="xbmc.json" version="7.0.0" />
+        <import addon="script.module.simplejson" version="2.0.10" />
```

Comment:
There is no need for importing simplejson anymore. I make use of ``import json`` in ``default.py``, which is actually a speed improvement for low-resource systems such as the Raspberry Pi.